bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -698,6 +701,55 @@ impl Room {
     pub fn enable_send_queue(&self, enable: bool) {
         self.inner.send_queue().set_enabled(enable);
     }
+
+    /// Store the given `ComposerDraft` in the state store using the current
+    /// room id, as identifier.
+    pub async fn save_composer_draft(&self, draft: ComposerDraft) -> Result<(), ClientError> {
+        Ok(self.inner.save_composer_draft(draft).await?)
+    }
+
+    /// Retrieve the `ComposerDraft` stored in the state store for this room.
+    pub async fn load_composer_draft(&self) -> Result<Option<ComposerDraft>, ClientError> {
+        Ok(self.inner.load_composer_draft().await?)
+    }
+
+    /// Remove the `ComposerDraft` stored in the state store for this room.
+    pub async fn clear_composer_draft(&self) -> Result<(), ClientError> {
+        Ok(self.inner.clear_composer_draft().await?)
+    }
+
+    /// Load the reply details for the given event id.
+    ///
+    /// This will return an `InReplyToDetails` object that contains the details
+    /// which will either be ready or an error.
+    pub async fn load_reply_details(
```

Comment:
Can we remove this one from this PR? I'd like to sit down with you to assess if we can do this differently, because I'm not quite satisfied even with the changes I added the other day…

Reply:
done! now the pr only contains draft saving , clearing and loading